release/e2e.py

```diff
@@ -1777,6 +1824,12 @@ def run_test(test_config_file: str,
         required=False,
         type=str,
         help="Name of the session to run this test.")
+    parser.add_argument(
+        "--app-config-id-override",
+        required=False,
+        type=str,
```

Comment:
Set the default as None?

Reply:
Ah, I saw you added in the function arg. It is up to you.

Reply:
The default value for the `default` is `None`, so not worth explicitly adding IMO: https://docs.python.org/3/library/argparse.html#default